docs/installation/local.md

```diff
@@ -2,169 +2,177 @@
 
 ## Dependencies required to run Orga Server
 
-* Python 3.7
-* Postgres
-* OpenSSL
+-   Python 3.7
+-   Postgres
+-   OpenSSL
 
-### For mac users
-```sh
-brew install postgresql
-````
-
-### For debian-based linux users
-```sh
-sudo apt-get update
-sudo apt-get install postgresql postgresql-contrib libssl-dev
-```
-
-## Steps
+## Get a copy of source code
 
-Make sure you have the dependencies mentioned above installed before proceeding further.
+- Clone the Open Event Server repository (from the development branch) and `cd ` into the directory.
 
-* **Step 0** - Clone the Open Event Server repository (from the development branch) and ```cd ``` into the directory.
 ```sh
-git clone -b development https://github.com/fossasia/open-event-server.git
+git clone https://github.com/fossasia/open-event-server.git
```

Comment:
It is important that only development branch gets cloned and not the gh-pages and other branches

Reply:
![image](https://user-images.githubusercontent.com/314607/95017049-16eb4f80-0681-11eb-8e65-2c43f64adb0d.png)

Without `-b` option, Git will choose the default branch, and this repo already set "development"as default branch.

Reply:
@hongquan It will clone all branches. Try running `git branch -a` after simply running git clone.

Reply:
@prateekj117 Agree. I will fix.